### PR TITLE
ci: using kaniko builder

### PR DESCRIPTION
**Pull request by albertmakan**

### CODE REVIEW

File: .github/workflows/develop-merged.yml [hunk 1]
```diff
@@ -10,9 +10,12 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - uses: ftrip-io/github-ci/build-and-push-to-dockerhub@ef2fd6ac85fe9e0ae63e5979a82a35bd5075bc33
+      - name: Kaniko build
+        uses: aevea/action-kaniko@master
```
Comment (panicmilos) [inline .github/workflows/develop-merged.yml]: We should not use @master. Let's use with hash of last commit.

File: .github/workflows/develop-merged.yml [hunk 2]
```diff
@@ -10,9 +10,12 @@ jobs:
...
         with:
+          image: ftripio/catalog-service-dev
+          path: ftrip.io.catalog-service
           username: ${{ secrets.DOCKER_USERNAME }}
```
Comment (panicmilos) [inline .github/workflows/develop-merged.yml:18]: Should we tag this one as the latest also?

File: .github/workflows/develop-merged.yml [hunk 3]
```diff
@@ -10,9 +10,12 @@ jobs:
...
           password: ${{ secrets.DOCKER_TOKEN }}
-          context: "ftrip.io.catalog-service"
-          tags: ftripio/catalog-service-dev:latest
+          cache: true
+          cache_registry: ftripio/catalog-service-cache
```
Comment (panicmilos) [inline .github/workflows/develop-merged.yml:21]: Did you create cache registry?

**Review by panicmilos — CHANGES_REQUESTED** [review]

File: .github/workflows/develop-merged.yml (hunk 1 above)
Comment (albertmakan) [reply .github/workflows/develop-merged.yml]: Or the tag name

File: .github/workflows/develop-merged.yml (hunk 2 above)
Comment (albertmakan) [reply .github/workflows/develop-merged.yml:18]: I think it tags it as the latest by default.

**Review by panicmilos — APPROVED** [review]

**Pull request merged**